### PR TITLE
Bump aiohttp across services, images, and packages

### LINKED ISSUE #3165: Deal with `aiohttp` and `cryptography` Dependabot PRs (December 2023)
As a Cal-ITP data user, I want our data infrastructure to use current package versions to avoid vulnerabilities and to avoid our dependencies from getting out of date and ultimately unsupported. 

We need to deal with a batch of dependabot PRs:
- [x] #3139 
- [x] #3140 
- [x] #3141 
- [x] #3142 
- [x] #3143 
- [x] #3144 
- [x] #3145 
- [x] #3146 
- [x] #3147 
- [x] #3148 
- [x] #3149 
- [ ] #3155

### PR DESCRIPTION
**Pull request by SorenSpicknall**
# Description

Bumps aiohttp across each relevant import, except in /airflow (which we try to keep in relative parity with the version deployed via Composer). Doesn't touch the `cryptography` import mentioned in #3165 for the same reason (parity for Composer in /airflow).

Resolves #3165 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?

Each relevant image built locally prior to merging. Some CI tasks will also build images.

## Post-merge follow-ups

- [x] No action required
- [ ] Actions required (specified below)

### CODE REVIEW

**Review by lauriemerrell — APPROVED** [review]
Technical changes LGTM, but nitpick that `airflow` is checked off for `aiohttp` in #3165 when not changed in this PR. 

For actions after merge, what would you plan to do with the two Airflow changes that aren't going to be made now -- would you just close those two dependabot PRs with a comment, or leave them open with a comment, or maybe make a new separate issue to start aggregating changes that we assume should be handled by the upcoming Airflow upgrade?

**Comment by SorenSpicknall** [comment]
> Technical changes LGTM, but nitpick that `airflow` is checked off for `aiohttp` in #3165 when not changed in this PR.
> 
> For actions after merge, what would you plan to do with the two Airflow changes that aren't going to be made now -- would you just close those two dependabot PRs with a comment, or leave them open with a comment, or maybe make a new separate issue to start aggregating changes that we assume should be handled by the upcoming Airflow upgrade?

Before I took on primary stewardship of this genre of task, it appears that previous maintainers tended to close accompanying Issues. I'm supportive of that general tact, since upgrades to Airflow itself naturally prompt upgrades to underlying dependencies and it's not particularly useful to track these small dependency changes in detail.

One thing I _will_ do is create an Issue to configure dependabot behavior to ignore certain parts of the repo, something we don't do by default - that way we're not prompted for updates to certain files that need to be manually maintained, like `Dockerfile.local` in the /airflow folder.

**Pull request merged**
